# HG changeset patch
# User Rik <rik@octave.org>
# Date 1442922445 25200
#      Tue Sep 22 04:47:25 2015 -0700
# Branch stable
# Node ID caa5de39147e296f545359d7a2ddcc3fae4e17d3
# Parent  c0566df8cde3bfe54812e31827881ea4abba895c
maint: Complete deprecation of gmap40 colormap.

* NEWS: Announce deprecation of function.

* image.txi: Remove gmap40 from manual.

diff --git a/NEWS b/NEWS
--- a/NEWS
+++ b/NEWS
@@ -253,16 +253,17 @@ Summary of important user-visible change
       ---------------------|------------------
       bicubic              | interp2
       delaunay3            | delaunay
       dump_prefs           | individual preference get/set routines
       find_dir_in_path     | dir_in_loadpath
       finite               | isfinite
       fmod                 | rem
       fnmatch              | glob or regexp
+      gmap40               | ----
       loadaudio            | audioread
       luinc                | ilu or ichol
       mouse_wheel_zoom     | mousewheelzoom axes property
       nfields              | numfields
       octave_tmp_file_name | tempname
       playaudio            | audioplayer
       saveaudio            | audiowrite
       syl                  | sylvester
diff --git a/doc/interpreter/image.txi b/doc/interpreter/image.txi
--- a/doc/interpreter/image.txi
+++ b/doc/interpreter/image.txi
@@ -191,25 +191,16 @@ between 0 and 1.
 @DOCSTRING(summer)
 
 @DOCSTRING(white)
 
 @DOCSTRING(winter)
 
 @DOCSTRING(contrast)
 
-An additional colormap is @code{gmap40}.  This code map contains only
-colors with integer values of the red, green and blue components.  This
-is a workaround for a limitation of gnuplot 4.0, that does not allow the color
-of line or patch objects to be set.  @code{gmap40} is chiefly useful to gnuplot
-4.0 users, and particularly in conjunction with the @var{bar}, @var{surf},
-and @var{contour} functions.
-
-@DOCSTRING(gmap40)
-
 The following three functions modify the existing colormap rather than
 replace it.
 
 @DOCSTRING(brighten)
 
 @DOCSTRING(spinmap)
 
 @DOCSTRING(whitebg)
